src/App.js

```diff
@@ -1,10 +1,41 @@
 import React, { Component } from 'react';
 
+// styles uses grid to have 2 columns, I think personally it looks better and easier to read
+//But again, what do I know what we need
+import './styles.css';
+
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <h1>Admin Portal</h1>
+        <h1 className="adminHeader">Admin Portal</h1>
+
+        <div className="divList grid">
+          <div className="divListElement">
```

Comment:
Let's rename this class to something like `grid-element`

Also, the naming convention for css classes is to use dashes like above